src/v2/System/Relay/createRelaySSREnvironment.ts

```diff
@@ -104,8 +98,8 @@ export function createRelaySSREnvironment(config: Config = {}) {
     }),
     relaySSRMiddleware.getMiddleware(),
     cacheMiddleware({
-      size: 2000, // max 2000 requests
-      ttl: 3600000, // 1 hour
+      size: Number(getENV("NETWORK_CACHE_SIZE")) ?? 2000, // max 2000 requests
+      ttl: Number(getENV("NETWORK_CACHE_TTL")) ?? 3600000, // 1 hour
```

Comment:
Can this use the defaults already implemented above rather than repeat them?

Reply:
Good q. Unfortunately no, not in this case. The defaults that are set through sharify apply to a running app, but the network layer is used extensively through our tests, which is independent of of sharify's bootstrapping. Hence the need to duplicate.

Reply:
Perhaps we can import the defaults directly from src/desktop/config.coffee? (Sorry if I missed something obvious.)

Reply:
Good suggestion @starsirius, that should work 👍